### ISSUE TITLE
How are the training set and test set divided?

### ISSUE THREAD

**Issue by wharstr0927** [962086838/Battery_fault_detection_NC_github#5]
For example, what are the respective proportions, vehicle IDs, and the distribution of positive and negative samples? Additionally, how can they be adapted to the following code snippet? 

> `class Dataset:
    def __init__(self, data_path, all_car_dict_path='../five_fold_utils/all_car_dict.npz.npy',
     ind_ood_car_dict_path='../five_fold_utils/ind_odd_dict1.npz.npy',
     train=True, fold_num=0):
        ind_ood_car_dict = np.load(ind_ood_car_dict_path, allow_pickle=True).item()
        self.ind_car_num_list = ind_ood_car_dict['ind_sorted']
        self.ood_car_num_list = ind_ood_car_dict['ood_sorted']
        self.all_car_dict = np.load(all_car_dict_path, allow_pickle=True).item()

        if train:
            car_number = self.ind_car_num_list[
                         :int(fold_num * len(self.ind_car_num_list) / 5)] + self.ind_car_num_list[
                                                                            int((fold_num + 1) * len(
                                                                                self.ind_car_num_list) / 5):]
        else:  # test
            car_number = self.ind_car_num_list[
                         int(fold_num * len(self.ind_car_num_list) / 5):int(
                             (fold_num + 1) * len(self.ind_car_num_list) / 5)] + self.ood_car_num_list

        self.data_path = data_path
        self.battery_dataset = []

        print('car_number is ', car_number)

        for each_num in car_number:
            for each_pkl in self.all_car_dict[each_num]:
                train1 = torch.load(each_pkl)
                self.battery_dataset.append(train1)`
